vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/certs/serving.go

```diff
@@ -57,6 +58,18 @@ type certKeyFileContent struct {
 	Key  []byte
 }
 
+func (c certKeyFileContent) CertSHA256() string {
+	h := sha256.New()
+	h.Write(c.Cert)
+	return fmt.Sprintf("%x", h.Sum(nil))
```

Comment:
you want this output consistent with output from must-gather inspect-certs so you can easily compare